packages/myst-to-react/src/basic.tsx

```diff
@@ -412,6 +417,15 @@ const BASIC_RENDERERS: BasicNodeRenderers = {
       </div>
     );
   },
+  root({ node, className }) {
+    if (node.children === undefined) {
+      return <></>;
+    }
+    const childComponents = node.children.map((child) => (
+      <MyST ast={child} className={className} />
```

Comment:
Any map will need a key. I think we can just pass the `node.children` to MyST, and that does the same as this. Will make an update.

Reply:
I've been reading a fair amount of React patterns lately, and still forgot `key`. Also, this is an obvious case for passing children directly ... not sure how I missed it.

Reply:
🚀